setup.py

```diff
@@ -25,6 +25,6 @@
       license='3-Clause BSD',
       packages=['splot'],
       include_package_data=True,
-      install_requires=['numpy', 'scipy', 'libpysal', 'mapclassify', 'palettable',
-                        'esda', 'pysal',],
+      install_requires=['numpy', 'libpysal', 'mapclassify', 'palettable',
+                        'esda', 'pysal','matplotlib','seaborn'],
```

Comment:
Do we want pysal and libpysal? Or, just libpysal?

Reply:
I can remove all `pysal` dependencies. they are mainly used in the `mapping.py` flies which I can also remove.

Reply:
I am using `ps.spreg.ols` to fit the moran line. For now I could just import spreg and use `spreg.ols` if that makes more sense. Then `pysal` can be removed?